Index: test_sample.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test_sample.py b/test_sample.py
new file mode 100644
--- /dev/null	(date 1638375325760)
+++ b/test_sample.py	(date 1638375325760)
@@ -0,0 +1,20 @@
+import pytest
+import main
+import pandas as pd
+import os
+
+files = 'files/'
+sample = pd.read_csv('sample_file.csv')
+sample = sample.astype(main.data_types)
+
+
+def test_file(dumple):
+    assert any(dumple.dtypes) == main.data_types
+
+
+for file in os.listdir(files):
+    if file.endswith(".csv"):
+        path = files + file
+        dumple = pd.read_csv(path)
+        dumple = main.fix(sample, dumple)
+        test_file(dumple)
